skbio/io/tests/test_fasta.py

```diff
@@ -435,12 +451,21 @@ def test_fasta_to_generator_invalid_files(self):
     def test_fasta_to_any_sequence(self):
         for constructor, reader_fn in ((Sequence,
                                         _fasta_to_biological_sequence),
-                                       (partial(DNA, validate=False),
-                                        _fasta_to_dna_sequence),
-                                       (partial(RNA, validate=False),
-                                        _fasta_to_rna_sequence),
-                                       (partial(Protein, validate=False),
-                                        _fasta_to_protein_sequence)):
+                                       (partial(DNA, validate=False,
```

Comment:
This is likely not the case but now that `lowercase` exists, can you drop `validate=False` on some of these?